README.md

```diff
@@ -1,24 +1,44 @@
-# Aqueduct Flood
+# Aqueduct Floods
 
-Aqueduct Flood is the third tool of the Aqueduct family. More info soon.
+## Getting Started
+
+1. Install Docker if you don't have it: https://hub.docker.com/editions/community/docker-ce-desktop-mac  Open it from Applications to get it to fire up.
+2. Clone the repo: https://github.com/greenriver/aqueduct-flood Todd forked it from the Vizzuality repo. Use the gr-develop branch for now.
+3. Cd into the repo and run docker compose up  to build your image and fire up the container. Visit localhost:3030. Ctrl C to stop.
+4. The app hangs up on API calls to Carto currently. If you want it to load without trying to access Carto, comment out this line in docker-compose.yml: REACT_APP_WRI_API_URL: //api.resourcewatch.org
+
+
+## Deployment
+
+1. Run `docker-compose run web yarn build` to compile.
+2. Copy the build to the wriorg aqueduct-gr branch build directory:
+  - Make sure you are on the aqueduct-gr branch in the wriorg repo.
+  - From aqueduct-flood root: `rsync -a --delete ~/Documents/aqueduct-flood/build/ ./floods/`. This will copy your build folder to the floods app in wriorg. The --delete flag is to make directories identical so that we don't keep old copies of the precache manifest around. As far as we know there is no reason to keep them.
```

Comment:
Could you write the rsync with both being relative paths as you would run from the floods checkout?

maybe like `rsync -a --delete ./build ../wriorg/application/aqueduct/floods/`. Check that second path, but something like that.

Reply:
If it's public, I'm not sure it makes sense to include that rsync command at all.